pipeline/5/Jenkinsfile-tier-1-master.groovy

```diff
@@ -0,0 +1,98 @@
+/*
+    Pipeline script for executing Tier 1 jobs for RH Ceph 5.0.
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def cephVersion = "pacific"
+def sharedLib
+def test_results = [:]
+def composeInfo = ""
+def tier1Jobs = ["rhceph-5-tier-1-deploy", "rhceph-5-tier-1-object", "rhceph-5-tier-1-rbd"]
+env.rhcephVersion = "5.0-rhel-8"
+
+// Pipeline script entry point
+node(nodeName) {
+
+	timeout(unit: "MINUTES", time: 30) {
+		stage('Validate tier1 Compose') {
+			checkout([
+                                $class: 'GitSCM',
+                                branches: [[name: '*/5.x_UMB']],
+                                doGenerateSubmoduleConfigurations: false,
+                                extensions: [[
+                                       $class: 'SubmoduleOption',
+                                       disableSubmodules: false,
+                                       parentCredentials: false,
+                                       recursiveSubmodules: true,
+                                       reference: '',
+                                       trackingSubmodules: false
+                                ]],
+                                submoduleCfg: [],
+                                userRemoteConfigs: [[url: 'https://github.com/manasagowri/cephci.git']]
+                        ])
+			script {
+				sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+				sharedLib.prepareNode()
+				def ciMessage = "${params.CI_MESSAGE}" ?: ""
+                                println "ciMessage : " + ciMessage
+
+                                if (ciMessage?.trim()) {
+                                        composeInfo = "${params.CI_MESSAGE}"
+                                }
+                                else {
+                                       	composeInfo = sharedLib.fetchTier1Compose()
+                                }
+			}
+		}
+	}
+
+	timeout(unit: "HOURS", time: 24) {
+		stage('Trigger tier1 Jenkins Jobs') {
+			script {
+				println "triggering jobs based on composeInfo"
+				println composeInfo
+				if(composeInfo){
```

Comment:
Let's avoid this check here as we know for sure the compose Info is set

Reply:
Made the necessary changes